docs/deployment/managed-scanning.md

```diff
@@ -0,0 +1,194 @@
+If you have already installed the public Semgrep GitHub app, skip this step and proceed to [Install the private Semgrep GitHub app](#install-the-private-semgrep-github-app).
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login).
+1. Click **<i class="fa-solid fa-gear"></i> Settings > Source Code Managers**.
+1. Click **Connect to GitHub**.
+1. Follow the steps to connect to your GitHub organization. Ensure that you grant access to the repositories you want to scan.
+
+### Install the private Semgrep GitHub app
+
+The private app must be installed by a **GitHub organization administrator**. If you are not a GitHub admin, an installation link is provided for you to share with your GitHub admin.
+
+1. Click **<i class="fa-solid fa-gear"></i> Settings > Source Code Managers**.
+1. Click **Register App**.
+![GitHub entry with public GitHub app connection](/img/scm-create-private-app.png)
+1. Follow the steps to install a private GitHub app in your org. Ensure that you enter your exact GitHub organization name and the correct type of GitHub account, typically **Organization**.
+1. Click **Register GitHub App**.
+1. If you are an admin on the GitHub organization, click **Continue**. Otherwise, share the provided link with your GitHub administrator.
+![Continue to SCM dialog](/img/scm-confirm-private-app.png#sm-width-noborder)
+1. Follow the prompts in GitHub to install the private app. Ensure that you grant access to the repositories you want to scan.
+
+:::caution Running into a 404?
+If you are brought to a GitHub 404 page, return to your [<i class="fas fa-external-link fa-xs"></i> GitHub Applications](https://github.com/settings/installations) page.
+:::
+
+### Completed installation
+
+A complete installation is displayed in the Source Code Manager entry as follows:
+
+![GitHub entry with public and private GitHub app connection](/img/zcs-code-access-enabled.png#md-width)
+_**Figure**. **Semgrep AppSec Platform > <i class="fa-solid fa-gear"></i> Settings > Source Code Managers** displaying a completed managed scanning set-up._
+
+You can also confirm a complete installation through your GitHub settings page, which should have two Semgrep apps:
+
+![GitHub settings page](/img/zcs-github-apps.png#bordered)
+_**Figure**. **GitHub > Settings > Applications** displaying both Semgrep apps. The private Semgrep app follows the convention **Semgrep Code - <span className="placeholder">YOUR_ORG_NAME</span>**_.
+
+## Add a repository
+
+1. In Semgrep AppSec Platform, click **<i class="fa-solid fa-folder-open"></i> Projects**.
+1. Click **Scan New Project > Semgrep Managed Scan**.
+1. Optional: If you can't find the repository you want to add, click **Can't find your project? > Sync projects**.
+1. Select the repositories you want to scan from the list.
+1. Click **Enable managed scanning**.
+
+- After enabling managed scanning, a full scan starts **immediately** on all the repositories you have added.
```

Comment:
@cgdolan What if a user enables say 100 or 1000 repos? The full scans are not started immediately but threaded out over time, right?

Reply:
They were indeed threaded for me